format/spec.md

```diff
@@ -951,24 +958,45 @@ Types are serialized according to this table:
+| **`long`**         | **`json long`**   | `1`                              |                                                                                                                                                                                                                                                                                                                           |
+| **`float`**        | **`json float`**  | `1.1`                            |                                                                                                                                                                                                                                                                                                                           |
+| **`double`**       | **`json double`** | `1.1`                            |                                                                                                                                                                                                                                                                                                                           |
+| **`decimal(P,S)`** | **`string`**      | `"0x3162"`                       | Stores the unscaled value, as the two's-complement big-endian binary using the minimum number of bytes, converted to a hexadecimal string prefixed by `0x`                                                                                                                                                                |
+| **`date`**         | **`json int`**    | `19054`                          | Stores days from the 1970-01-01                                                                                                                                                                                                                                                                                           |
+| **`time`**         | **`json long`**   | `36000000000`                    | Stores microseconds from midnight                                                                                                                                                                                                                                                                                         |
+| **`timestamp`**    | **`json long`**   | `1646277378000000`               | Stores microseconds from 1970-01-01 00:00:00.000000                                                                                                                                                                                                                                                                       |
```

Comment:
I think we need to specify UTC here as well, we won't be storing the default with timezone information even if the timestamp type has it.